tests/integration/corrupt-dump.tcl

```diff
@@ -173,7 +173,8 @@ foreach sanitize_dump {no yes} {
             verify_log_message 0 "*skipping empty key: hash_ziplist*" 0
             verify_log_message 0 "*skipping empty key: zset*" 0
             verify_log_message 0 "*skipping empty key: zset_ziplist*" 0
-            verify_log_message 0 "*empty keys skipped: 8*" 0
+            verify_log_message 0 "*skipping empty key: zset_listpack*" 0
```

Comment:
are there really both types of encoded zsets in the same rdb file? how did you generate it?

Reply:
I used `debug populate` to generate it, and modified the saved code.

```c
        // zset ziplist
        robj *o = createObject(OBJ_ZSET,ziplistNew());
        o->encoding = OBJ_ENCODING_LISTPACK;
        dbAdd(c->db,createStringObject("zset_ziplist",12),o);

        // zset ziplist
        dbAdd(c->db,createStringObject("zset_listpack",13),createZsetListpackObject());
```